### ISSUE TITLE
Bug: error:-83 

### ISSUE THREAD

**Issue by linuxrecon** [ONLYOFFICE/DocumentServer#443]
We want to migrate our documents to an OnlyOffice (Document) Server. We tried a few documents which some of them are showing up flawlessly while some of them don't. The following error message appears in the browser (after waiting almost 2 minutes where OnlyOffice tries to open the file): "Timeout while converting". Output from converter/out.log for e.g. is:

[2018-12-06 12:06:42.650] [ERROR] nodeJS - stdout (id=_DV3JztGHzvdG7GTSjk_):
[2018-12-06 12:06:42.650] [ERROR] nodeJS - stderr (id=_DV3JztGHzvdG7GTSjk_):
[2018-12-06 12:06:42.650] [ERROR] nodeJS - ExitCode (code=0;signal=SIGTERM;error:-83;id=_DV3JztGHzvdG7GTSjk_)

All documents producing this error are .docx files made with the latest MS Office suite. LibreOffice is opening the files without any problems albeit the format is screwed up.

I also tried the desktop editors which are having the exact same behaviour and error message. Even an older version of the document server (5.1.5.59, currently running the 5.2.3.64 within Docker) is giving the exact same error.

As already mentioned, it's a Docker setup running under a updated Debian 9 installation. Tested with Chromium and Firefox browser at the client.

If you need an example document, please let me know and I will try to do my best to send you one. The documents we're experiencing this issue with are containing sensible data and I'm unable to send them to you right now.

**Comment by ShockwaveNN** [comment]
@linuxrecon Hi, we definitely need this documents. You can try to replace all sensible data with gibberish, but try to keep document structure the same

**Comment by linuxrecon** [comment]
@ShockwaveNN Thanks for your answer. I tried to get the "ok" from the authorities to send you the documents. However, they refused even though I said that we would replace everything with gibberish data.

I did a little bit more research and encountered that these documents contain drawings which are made with MS Visio which then were exported as a JPG, PNG or whatsoever. These MS Visio exported drawings having a file size of round about 500kb each. If I simply copy one of these drawings into a new docx document, OnlyOffice is unable to open this file later on as well. Reducing the image quality with the help of a image editor (which brings down the file size to 89kb) results in OnlyOffice being able to open the test document flawlessly. Removing all MS Visio drawings from the document also resulted in OnlyOffice being able to open the document.
I also tried a different picture from the web with a resolution of 1920x1080 and a file size of round about 600kb. However, OnlyOffice is capable of opening a docx file with this picture included. So it seems that the MS Visio drawings are the problem...

**Comment by ShockwaveNN** [comment]
@linuxrecon we just checked MS Visio 2016 and cannot reproduce your bug.
Could you specify exact version of MS Visio and possible make some non-private Visio file with which problem occurs?

**Comment by linuxrecon** [comment]
@ShockwaveNN I've tested the Visio export as well. To be more specific, I tried to copy & paste a Visio picture to a docx Word document, saved the file and uploaded it to nextcloud. I was then able to open the file with OnlyOffice.
Sadly I'm not allowed to send you the original file where the error still persist. I found other issues in displaying simple Word files as well. Unfortunately I didn't got the permission to share this will with you as well ... If I ever get a permission to do so, I open a new bug here. So far, thanks for your help. I close this bug for now.
